### ISSUE TITLE
Evaluating the detection results

### ISSUE THREAD

**Issue by Guanshuyan** [DongshuoYin/garbage_dump_detection#4]
Hello, I'd like to ask how the model determines the success of garbage dump detection. Is it using Intersection over Union (IoU)?

**Comment by DongshuoYin** [comment]
Yes, the model treats the object as a garbage dump when IoU is over 50%. See the Supplementary Material to the paper for more details.

**Comment by Guanshuyan** [comment]
> Yes, the model treats the object as a garbage dump when IoU is over 50%. See the Supplementary Material to the paper for more details.

Thank you very much! By the way, I want to use BCA-NET to detect other categories of satellite image data, and whether it can be trained directly after adjusting the format of the data set to be consistent

**Comment by DongshuoYin** [comment]
Sure, you can learn the data format of garbage dump data set and adjust your own pictures and labels to the same way.



Yin

---Original---
From: "Shuyan ***@***.***&gt;
Date: Fri, Nov 10, 2023 16:13 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [DongshuoYin/garbage_dump_detection] Evaluating the detectionresults (Issue #4)




  
Yes, the model treats the object as a garbage dump when IoU is over 50%. See the Supplementary Material to the paper for more details.
  
Thank you very much! By the way, I want to use BCA-NET to detect other categories of satellite image data, and whether it can be trained directly after adjusting the format of the data set to be consistent
